src/base1/cockpit.js

```diff
@@ -99,7 +99,7 @@ function invoke_functions(functions, self, args) {
 /* -------------------------------------------------------------------------
  * Channels
  *
- * Public: https://files.cockpit-project.org/guide/api-cockpit.html
+ * Public: http://cockpit-project.org/guide/latest/api-cockpit.html
```

Comment:
This page is a bit thin and not really that useful -- Depending on what the purpose of that comment is, might it be better to refer to http://cockpit-project.org/guide/latest/development.html which lists and links to the complete API spec?